HU-73: OroAkeneoBundle Upgrade to 6.0

StevensonKuo opened the pull request:
- checkthe functions and it could work now, but not giving prompt that there is no bug.

Commit:
HU-73 Upgrade to 6.0
 - checkthe functions and it could work now, but not giving prompt that there is no bug.

Commit:
HU-73: Upgrade to 6.0
 - format updated, then test.

Commit:
HU-73: OroAkeneoBundle Upgrade to 6.0
 - basically done.

Commit:
HU-73: OroAkeneoBundle Upgrade to 6.0
 - finalize

Commit:
HU-73: OroAkeneoBundle Upgrade to 6.0
 - Improvement/avoid loading proxy relations data

StevensonKuo commented on ImportExport/Strategy/StrategyRelationsTrait.php:
```diff
@@ -56,16 +49,27 @@ protected function updateRelations($entity, array $itemData = null)
+                                $entityName,
+                                $fieldName,
+                                $isPersistRelation
+                            ),
```
apply from:
https://github.com/creativestyle/OroAkeneoBundle/pull/5/files

Commit:
HU-73: OroAkeneoBundle Upgrade to 6.0
 - StrategyValidationTrait might be used in class without doctrine attribute.

alex-n-2k7 commented on Resources/config/oro/routing.yml:
```diff
@@ -1,4 +1,4 @@
 akeneo_integration_shipping:
-  resource:     "@OroAkeneoBundle/Controller/ValidateConnectionController.php"
-  type:         annotation
-  prefix:       /akeneo-validate
```
please add empty line at the end of file

StevensonKuo replied:
Done

Commit:
HU-19 OroAkeneoBundle
 - fixed exceeds line
 - fixed pass null to strtolower
 - add empty line to yaml file
 - for Migrations/Schema/v1_14/OroAkeneoMigration.php I think there is an accident semicolon will break the SQL in mysql platform.

Commit:
HU-73 Upgrade to 6.0
 - remove sync products field.
